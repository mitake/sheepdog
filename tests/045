#!/bin/bash

# Test partial vdi read and write

seq=`basename $0`
echo "QA output created by $seq"

here=`pwd`
tmp=/tmp/$$
status=1        # failure is the default!

# get standard environment, filters and checks
. ./common.rc
. ./common.filter

_cleanup

for i in 0 1 2; do
    _start_sheep $i
done

_wait_for_sheep 3

$COLLIE cluster format -c 2
$COLLIE vdi create test 4M

for i in `seq 0 7`; do
    echo $i | $COLLIE vdi write test $(($i * 1536 + 512)) 512
done

for i in `seq 0 7`; do
    $COLLIE vdi read test $(($i * 1536 + 512)) 512 | md5sum
done

$COLLIE vdi read test | md5sum
