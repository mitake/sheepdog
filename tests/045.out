QA output created by 045
using backend farm store
e0b27e7466a3c21d0a4dedfed8bb9184  -
f35835c0a25be5ee75a536d1816c1db4  -
0faf5f38c28a38a6db1e6dfcdf259141  -
83bffbfb00dbcb7d6b4a2fa9274175b9  -
a8775e30ddc5eda14d76e5361a514392  -
8124234db4755d4772c3440f2cbd1ecb  -
62630992718e77c73873ce1672d80786  -
19b9d4d75a5a71bfad364628880fa913  -
2471b1f99fe89885e2267c5d4f94723b  -
